mentorship ios/Models/RequestModel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  RequestModel.swift
+//  Created on 24/07/20
+//  Created for AnitaB.org Mentorship-iOS 
+//
+
+struct RequestsList: Decodable {
+    let sent: Sent?
+    struct Sent: Decodable {
+        let accepted: [RequestStructure]?
+        let rejected: [RequestStructure]?
+        let completed: [RequestStructure]?
+        let cancelled: [RequestStructure]?
+        let pending: [RequestStructure]?
+    }
+    let received: Received?
+    struct Received: Decodable {
```

Comment:
is this not the same structure as Sent? Can't we reuse this code?

Reply:
We can. My reason for not creating an additional structure was that-
1. These are just 5 lines of code. And, writing like this seemed better due to future-proofing of potential differences between sent and received structure.
2. Creating a new structure and reusing would create another layer of nesting. It is already very heavily nested.
3. Together, they didn't formed a very logical block.